libs/eo/landing-page/shell/src/lib/layout/eo-landing-page-call-to-action.component.ts

```diff
@@ -33,21 +33,50 @@ const selector = 'eo-landing-page-call-to-action';
 
       ${selector} {
         display: block;
-        margin-top: var(--watt-space-xl);
         text-align: center;
+      }
+      .${selector}__h2 {
+        @include watt.typography-watt-headline-2; // This overrides the styles applied from Angular Material on h2 tags
+        text-transform: none; // This overrides the uppercased styling from watt
+        display: inline-block;
+      }
+      .${selector}__call-to-action-wrapper {
+        height: calc(75 * var(--watt-space-xs));
+        background: var(
+          --watt-color-focus-selection
+        ); // This is the light-blue-ish background color
+        display: flex;
+        align-items: center;
+        justify-content: center;
+      }
+      .${selector}__link-icon {
+        width: 24px;
+        height: 24px;
+        margin-left: calc(2.5 * var(--watt-space-xs));
+        cursor: pointer;
+      }
 
-        .${selector}__h2 {
-          @include watt.typography-watt-headline-2; // This overrides the styles applied from Angular Material on h2 tags
-          text-transform: none; // This overrides the uppercased styling from watt
-        }
+      .${selector}__login-button > a {
+        width: calc(40 * var(--watt-space-xs));
       }
     `,
   ],
   template: `
-    <eo-landing-page-column-layout [layoutType]="'full'">
-      <h2 class="${selector}__h2">Log in with your company NemID</h2>
-      <eo-landing-page-login-button></eo-landing-page-login-button>
-    </eo-landing-page-column-layout>
+    <div class="${selector}__call-to-action-wrapper">
+      <eo-landing-page-column-layout [layoutType]="'full'">
+        <h2 class="${selector}__h2">Log in with your company NemID</h2>
+        <a href="https://www.nemid.nu/dk-en/about_nemid/index.html">
+          <img
+            class="${selector}__link-icon"
+            src="/assets/images/icons/primary-info-icon.svg"
+            alt="EnergyOrigin NemID log in"
+          />
```

Comment:
I think aligning the icon vertically so the center equals the center of the text could improve the look of this, as currently it seems a little off to me. 
![chrome_2022-03-11_12-25-15](https://user-images.githubusercontent.com/2674493/157858422-eefc3446-4dd7-4fe2-a332-1280023c6654.png)